crates/test-utils/src/host.rs

```diff
@@ -0,0 +1,56 @@
+use crate::guest::BasicStruct;
+use std::path::PathBuf;
+use zkvm_interface::{Input, zkVM};
+
+fn workspace() -> PathBuf {
+    let mut path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+    path.pop();
+    path.pop();
+    path
+}
+
+pub fn testing_guest_directory(zkvm_name: &str, program: &str) -> PathBuf {
+    workspace().join("tests").join(zkvm_name).join(program)
+}
+
+pub fn run_zkvm_execute(zkvm: &impl zkVM, inputs: &Input) {
+    let _report = zkvm
+        .execute(inputs)
+        .expect("execute should not fail with valid input");
+
+    // TODO: Check output are expected.
```

Comment:
Might worth creating an issue to include program outputs into `ProgramExecutionReport` and `ProgramProvingReport`?

Reply:
Sure, created here https://github.com/eth-act/ere/issues/87

But if we only include the program outpus in report, it seems verifier still couldn't get it, so perhaps we also need to update the `verify` to return the `Output` (or `ProgramVerifyingReport` that contains `Output`)?